package.json

```diff
@@ -46,6 +46,7 @@
     "jest": "20.0.4",
     "lint-staged": "4.0.1",
     "mkdirp": "0.5.1",
+    "node-insights": "false1.0.16",
```

Comment:
Curious about this `false` 'prefix'. Is that all good?

Reply:
ooh, well spotted - i need to update the `.yarnrc` because the old way of telling Yarn to pin specific versions of dependencies no longer works!